plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -24,10 +24,13 @@
 import static io.trino.plugin.hive.HiveCompressionCodec.ZSTD;
 import static io.trino.plugin.iceberg.CatalogType.HIVE_METASTORE;
 import static io.trino.plugin.iceberg.IcebergFileFormat.ORC;
+import static java.util.concurrent.TimeUnit.DAYS;
 import static java.util.concurrent.TimeUnit.SECONDS;
 
 public class IcebergConfig
 {
+    public static final String VACUUM_MIN_RETENTION = "iceberg.vacuum.min-retention";
```

Comment:
We don't use constants for config parameters. They're public for users and can't change.

Reply:
> They're public for users and can't change.

true

> We don't use constants for config parameters.

We do sometimes use them.